src/CSSList/CSSBlockList.php

```diff
@@ -30,10 +30,8 @@ public function __construct($iLineNo = 0)
 
     /**
      * @param array<int, DeclarationBlock> $aResult
-     *
-     * @return void
      */
-    protected function allDeclarationBlocks(array &$aResult)
+    protected function allDeclarationBlocks(array &$aResult): void
```

Comment:
Can't help thinking that these `all...` methods should perhaps be returning the result, rather than having it as a parameter passed by reference - unless the array is supposed to be merged with some original array - but that is beyond the scope of this PR.

Reply:
Looked at the code, and these functions call each other to build up the array, so it does need to be a parameter passed by reference.